include/openthread/border_routing.h

```diff
@@ -103,6 +108,7 @@ typedef struct otBorderRoutingRouterEntry
     bool         mStubRouterFlag : 1;           ///< The router's Stub Router flag.
     bool         mIsLocalDevice : 1;            ///< This router is the local device (this BR).
     bool         mIsReachable : 1;              ///< This router is reachable.
+    bool         mIsPeerBr : 1;                 ///< This router is (likely) a peer BR connected to same mesh.
```

Comment:
```suggestion
    bool         mIsPeerBr : 1;                 ///< This router is (likely) a peer BR connected to the same Thread mesh.
```

For consistency with the command documentation

Reply:
I shortened it to "This router is (likely) a peer BR" to avoid going over line limit here and since this is explained in detail in the `otBorderRoutingRouterEntry` description.